### PR TITLE
Add hull theme

### PR DESCRIPTION
**Pull request by fami-fish**

### CODE REVIEW

File: packages/hull [hunk 1]
```diff
@@ -0,0 +1,4 @@
+type = theme
+repository = https://github.com/nick-1666/hull.git
+maintainer = Nick-1666 admin@nick-1666.dev
```
Comment (scorphus) [inline packages/hull:3]:
```suggestion
maintainer = Nick-1666 <admin@nick-1666.dev>
```

**Review by scorphus — COMMENTED** [review]
Please consider my suggestion below, besides another to add a screenshot to the theme's README.md.

**Comment by nick-1666** [comment]
Added screenshot and fixed format

**Comment by faho** [comment]
@scorphus This is failing the tests because the repo has issues disabled.

That means users won't have anywhere to report problems.

Either this PR should be rejected, the issues opened or the rules would have to be changed to allow for issues being closed.

**Pull request closed without merging**